src/main/java/com/tvtracker/enterprise/TvTrackerController.java

```diff
@@ -87,7 +86,8 @@ public ResponseEntity signUpUser(@RequestBody UserAccount userAccount) {
      * @return a valid user token for session authentication
      */
     @GetMapping("/authenticate")
-    public ResponseEntity authenticateUser(@RequestParam(value="username", required=true) String username, @RequestParam(value="password", required=true) String password) {
+    public ResponseEntity authenticateUser(@RequestParam(value="username", required=true) String username,
+                                           @RequestParam(value="password", required=true) String password) {
```

Comment:
Change the layout so that It is easier to see all the request params so you don't have to scroll to see them all .